### ISSUE TITLE
[$20]{MSFT-121} Sample challenge page: The link under the heading 'eligible events' doesn't have a label indicating its purpose.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2819]
**Steps to Reproduce**
1. Go to the sample challenge page: https://clicktime.symantec.com/34CaebazimgmFxdkBZnihRc7Vc?u=https%3A%2F%2Fwww.topcoder.com%2Fchallenges%2F<sample-Sample challenge page:challengeid>?tab=details. 2. Ensure the screen reader is switched on. 3. Navigate to the heading 'eligible events'. 4. Press the down arrow key. 5. Notice that JAWS reads 'link graphic and a series of numbers. 

**Expected Result**
The link should have a meaningful link text or label.

**Actual Result**
Screen reader reads the graphic link and a series of numbers and characters which are not an appropriate link text.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.4 Link Purpose (In Context)
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2019

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096018 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096018 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096018 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2819 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2819 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2819 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2819 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097313 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097313 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097313 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
@veshu @mishacucicea  @sushilshinde @lakshmiathreya   I am unable to test it on test server , need your help to setup for test scenario. 
https://monosnap.com/file/zeddu3L3bUhtd5U4B1UGrsxO9D8Z7Y

**Comment by lakshmiathreya** [comment]
@Oanh-and-only-Oanh - pls note - we are unable to test this Issue with the test urls.

**Comment by veshu** [comment]
@drasticdpk @lakshmiathreya I am trying to create the challenge. give me an hour on this. thanks!

**Comment by lakshmiathreya** [comment]
@veshu Pls let us know the challenge created with event to test this. @drasticdpk fyi...

**Comment by veshu** [comment]
@lakshmiathreya @drasticdpk We are still not be able to add eligible events on dev environment. 

I will get back as soon as I got help. Thanks!

**Comment by veshu** [comment]
@drasticdpk @lakshmiathreya Is this challenge https://www.topcoder-dev.com/challenges/30041559 works?

**Comment by drasticdpk** [comment]
@veshu  which credential  are you using ? I am unable from ToneyJ. Thanks FYI @lakshmiathreya

**Comment by lakshmiathreya** [comment]
We will need to test this on Prod directly - @sushilshinde @nkumar-topcoder @glenn - perhaps a we should have a second code review and merge to Prod if second review is fine - since QA on Dev is not possible. fyi @nithyaasworld @drasticdpk @tosha5252

**Comment by drasticdpk** [comment]
Verified on Windows/Chrome/JAWS.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097313<br/><br/>```This is an automated message for crazyk via Topcoder X```
